src/page/index.ts

```diff
@@ -0,0 +1,10 @@
+export { MenuContentPage } from './menu-content.page';
+export { AddressStepPage } from './address-step.page';
+export { BankPaymentPage } from './bank-payment.page';
+export { ProductAddedPage } from './product-added-modal.page';
+export { ProductListPage } from './product-list.page';
+export { SummaryContentPage } from './summary-step.page';
+export { SignInContent } from './sign-in-step.page';
```

Comment:
add the page suffix: SignInPage or SignInStepPage, that implies renaming the class name.